src/app/app.module.ts

```diff
@@ -4,10 +4,11 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { CoreModule } from './core/core.module';
 import { LayoutModule } from './layout/layout.module';
+import { HomeModule } from './home/home.module';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [CoreModule, AppRoutingModule, LayoutModule],
+  imports: [CoreModule, AppRoutingModule, LayoutModule, HomeModule],
```

Comment:
It's not necessary to add homemodule to the appmodule since we are launching it with lazy loading